core/lib/types/src/snapshots.rs

```diff
@@ -3,28 +3,47 @@ use zksync_basic_types::{L1BatchNumber, MiniblockNumber};
 
 use crate::{commitment::L1BatchWithMetadata, Bytes, StorageKey, StorageValue};
 
+/// Information about all snapshots persisted by the node.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct AllSnapshots {
+    /// L1 batch numbers for complete snapshots. Ordered by descending number (i.e., 0th element
+    /// corresponds to the newest snapshot).
     pub snapshots_l1_batch_numbers: Vec<L1BatchNumber>,
+    /// Same as `snapshots_l1_batch_numbers`, but for incomplete snapshots (ones in which some
+    /// storage log chunks may be missing).
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
```

Comment:
Do I understand correctly that it removes this field completely from response? Is removing some fields, i.e. dynamic schema in any way standard behaviour? I would just expect the field to always be there, even if empty

Reply:
Also, why do we want to have this field in the first place?